LICENSE

```diff
@@ -1,7 +1,7 @@
-Copyright 2017 Alex Jover Morales, @alexjoverm <alexjovermorales@gmail.com>
+Copyright 2017 @--username-- <--usermail-->
```

Comment:
Okay, I tryed to leave LICENSE as it was. I agree, we are used to get GitHub username. It could be literally the same as Twitter :) but we cannot assure